contents/docs/deployment/securing-posthog.md

```diff
@@ -40,3 +40,22 @@ Secret keys are used to encrypt cookies and password reset emails, [among other
 ```bash
 openssl rand -hex 32
 ```
+
+### Secret Key with Docker Compose
+
+When using Docker Compose, you will need to manually update the `docker-compose.yml` file with a secret key that is unique to your instance.
+
+**⚠️ Note: Do not use our placeholder key! Read more about the importance of this key [here](/docs/deploymentsecuring-posthog).**
```

Comment:
I've created an issue https://github.com/PostHog/posthog/issues/1404 to ask for this to be more obvious for users